### PR TITLE
Fix some issues detected by go vet/golint

### PR DESCRIPTION
**Pull request by pdalpra**
This PR fixes some issues that got detected by `go vet`/`golint`:

* Usage of `\n` in `fmt.Println`
* Acronyms in names not in full caps
* `return` in `else` blocks
* unused second values in `for` loops

`golint` also states that anything exported should be documented, but I'll to dig deeper in the codebase to provide helpful comments :)

### CODE REVIEW

File: crane/container.go [hunk 1]
```diff
@@ -69,15 +69,15 @@ type RunParameters struct {
 	RawCapAdd       []string    `json:"cap-add" yaml:"cap-add"`
 	RawCapDrop      []string    `json:"cap-drop" yaml:"cap-drop"`
 	RawCgroupParent string      `json:"cgroup-parent" yaml:"cgroup-parent"`
-	CpuPeriod       int         `json:"cpu-period" yaml:"cpu-period"`
```
Comment (michaelsauter) [inline crane/container.go]: I'm not entirely sure, but I think I read at some point that gaoling prefers camel case here. I definitely think that it is easier to read then the alternativ versions. Mind removing all of those changes?

File: crane/container.go [hunk 2]
```diff
@@ -494,32 +492,32 @@ func (r *RunParameters) Cmd() []string {
...
 
-func (c *container) Id() string {
+func (c *container) ID() string {
 	if len(c.id) == 0 && !c.Unique() {
 		// `docker inspect` works for both image and containers, make sure this is a
-		// container payload we get back, otherwise we might end up getting the Id
+		// container payload we get back, otherwise we might end up getting the ID
 		// of the image of the same name.
-		c.id = inspectString(c.ActualName(), "{{if .State}}{{.Id}}{{else}}{{end}}")
+		c.id = inspectString(c.ActualName(), "{{if .State}}{{.ID}}{{else}}{{end}}")
```
Comment (michaelsauter) [inline crane/container.go]: I think this is actually `Id` in the returned JSON from Docker.

Comment (pdalpra) [inline crane/container.go]: woops, my bad !

File: crane/container.go (hunk 1 above)
Comment (pdalpra) [inline crane/container.go]: This is actually a rule from `golint`, taken from : https://github.com/golang/go/wiki/CodeReviewComments#initialisms
But I can remove them if you prefer :)

Comment (michaelsauter) [inline crane/container.go]: Oh. I must have confused it then with something else. I guess it's better to follow the guideline then, so leave it in.

Thanks for pointing me to the document.

Comment (pdalpra) [inline crane/container.go]: My pleasure :)

File: crane/container.go [hunk 3]
```diff
@@ -1024,8 +1021,8 @@ func (c *container) prefixedName() string {
 }
 
 // Return the image id of a tag, or an empty string if it doesn't exist
-func imageIdFromTag(tag string) string {
-	args := []string{"inspect", "--format={{.Id}}", tag}
+func imageIDFromTag(tag string) string {
+	args := []string{"inspect", "--format={{.ID}}", tag}
```
Comment (michaelsauter) [inline crane/container.go]: This should stay `Id`, too :)

Comment (pdalpra) [inline crane/container.go]: fixed !

**Comment by pdalpra** [comment]
Everything should be okay now, just rebased my PR against master :)

**Comment by michaelsauter** [comment]
Thank you!

**Pull request merged**